Dockerfile

```diff
@@ -2,26 +2,30 @@
 FROM golang:1.14 as builder
 
 WORKDIR /workspace
-# Copy the Go Modules manifests
+
+# copy modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
+
+# cache modules
 RUN go mod download
 
-# Copy the go source
+# copy source code
 COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
 
-# Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o manager main.go
+# build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build -a -o image-reflector-controller main.go
+
+FROM alpine:3.12
```

Comment:
alpine because useful for trouble-shooting?

Reply:
For CVE scanning as the distroless is not support by most scanners and we want something like Xray that can lookup the go modules and OS packages.

Reply:
Another argument is consistency across the toolkit. Having the same base image with the same OS packages means that we could automate the base image update for all controllers.

Reply:
Ah I forgot about docker buildx and multi-arch builds.. if we want ARM and ARM64 then we have to use the alpine multi-arch base image or another image like Debian.

Reply:
Didn't think of the CVE scanning thing, that's a good reason :+1: